Library/Homebrew/tab.rb

```diff
@@ -250,6 +250,10 @@ def to_s
     when true  then s << "Poured from bottle"
     when false then s << "Built from source"
     end
+    unless time.nil?
+      s << "Installed" if s.empty?
+      s << Time.at(time).strftime("on %Y-%m-%d at %H:%M:%S")
```

Comment:
Could we use one of Ruby's default time outputs? Bonus points if it's localised.